src/components/info-sheet/info-sheet.html

```diff
@@ -623,5 +623,14 @@
       <span sheet-text translate>When your transaction is confirmed, the multisig wallet will show up in the wallet list. You can check the transaction status in the wallet history.</span>
       <span sheet-button-text translate>OK</span>
     </info-sheet-template>
+
+    <info-sheet-template *ngSwitchCase="'exchange-crypto-disclaimer'" type="info">
+      <span sheet-title-text translate>Swap Crypto</span>
+      <span sheet-text>
+        The Swap Crypto function is provided by Changelly and 1Inch, and subject to the service providers' terms and conditions, which you accept by using the service. Use of the Swap Crypto function is solely between you and the service providers.
```

Comment:
If this are services "Changelly", "1Inch", It would be better if they're emphasized or a link to their domain would be much better.